exporters/otlp/otlptrace/internal/otlpconfig/options_test.go

```diff
@@ -92,10 +92,30 @@ func TestConfigs(t *testing.T) {
 		{
 			name: "Test With Endpoint",
 			opts: []otlpconfig.GenericOption{
-				otlpconfig.WithEndpoint("someendpoint"),
+				otlpconfig.WithEndpoint("https://127.0.0.1:1234"),
 			},
 			asserts: func(t *testing.T, c *otlpconfig.Config, grpcOption bool) {
-				assert.Equal(t, "someendpoint", c.Traces.Endpoint)
+				assert.Equal(t, "127.0.0.1:1234", c.Traces.Endpoint)
+			},
+		},
+		{
+			name: "Test With Endpoint without scheme",
+			opts: []otlpconfig.GenericOption{
+				otlpconfig.WithEndpoint("127.0.0.1:4318"),
+			},
+			asserts: func(t *testing.T, c *otlpconfig.Config, grpcOption bool) {
+				assert.Equal(t, "127.0.0.1:4318", c.Traces.Endpoint)
+				assert.Equal(t, otlpconfig.DefaultTracesPath, c.Traces.URLPath)
+			},
+		},
+		{
+			name: "Test With Endpoint with upper case scheme",
+			opts: []otlpconfig.GenericOption{
+				otlpconfig.WithEndpoint("HTTPS://127.0.0.1:4318"),
+			},
+			asserts: func(t *testing.T, c *otlpconfig.Config, grpcOption bool) {
+				assert.Equal(t, "127.0.0.1:4318", c.Traces.Endpoint)
+				assert.Equal(t, otlpconfig.DefaultTracesPath, c.Traces.URLPath)
 			},
```

Comment:
Could you add a test with no scheme and the config set as insecure?

Reply:
And one with an invalid URL? (adding a space at the beginning of the URL would make it unparsable for example)